data/multiplayer/scenarios/4p_Isars_Cross.cfg

```diff
@@ -92,9 +96,103 @@
         # not clearing the variable (if set), leaving it for add-ons
     [/event]
 
+    #waterfall animation
     [terrain_graphics]
+        [tile]
+            pos=1
+            type=Xv^Xxxx
+        [/tile]
         [image]
             layer=0
+            name=map-specific/isar-waterfall/01.webp:42,map-specific/isar-waterfall/02.webp:42,map-specific/isar-waterfall/03.webp:42,map-specific/isar-waterfall/04.webp:42,map-specific/isar-waterfall/05.webp:42,map-specific/isar-waterfall/06.webp:42,map-specific/isar-waterfall/07.webp:42,map-specific/isar-waterfall/08.webp:42,map-specific/isar-waterfall/09.webp:42,map-specific/isar-waterfall/10.webp:42,map-specific/isar-waterfall/11.webp:42,map-specific/isar-waterfall/12.webp:42,map-specific/isar-waterfall/13.webp:42,map-specific/isar-waterfall/14.webp:42,map-specific/isar-waterfall/15.webp:42,map-specific/isar-waterfall/16.webp:42,map-specific/isar-waterfall/17.webp:42,map-specific/isar-waterfall/18.webp:42,map-specific/isar-waterfall/19.webp:42,map-specific/isar-waterfall/20.webp:42,map-specific/isar-waterfall/21.webp:42,map-specific/isar-waterfall/22.webp:42,map-specific/isar-waterfall/23.webp:42,map-specific/isar-waterfall/24.webp:42,map-specific/isar-waterfall/25.webp:42,map-specific/isar-waterfall/26.webp:42,map-specific/isar-waterfall/27.webp:42,map-specific/isar-waterfall/28.webp:42,map-specific/isar-waterfall/29.webp:42,map-specific/isar-waterfall/30.webp:42,map-specific/isar-waterfall/31.webp:42,map-specific/isar-waterfall/32.webp:42,map-specific/isar-waterfall/33.webp:42,map-specific/isar-waterfall/34.webp:42,map-specific/isar-waterfall/35.webp:42,map-specific/isar-waterfall/36.webp:42,map-specific/isar-waterfall/37.webp:42,map-specific/isar-waterfall/38.webp:42,map-specific/isar-waterfall/39.webp:42,map-specific/isar-waterfall/40.webp:42,map-specific/isar-waterfall/41.webp:42,map-specific/isar-waterfall/42.webp:42,map-specific/isar-waterfall/43.webp:42,map-specific/isar-waterfall/44.webp:42,map-specific/isar-waterfall/45.webp:42,map-specific/isar-waterfall/46.webp:42,map-specific/isar-waterfall/47.webp:42,map-specific/isar-waterfall/48.webp:42
```

Comment:
Can't this use progressive notation?

`map-specific/isar-waterfall/[01~48].webp:42`

Reply:
That's a good point.